### PR TITLE
Fix cherry-pick link

### PR DESCRIPTION
**Pull request by simonswine**
Relative links points to wrong path

### CODE REVIEW

**Comment by guineveresaenger** [comment]
Thank you for catching that!
/lgtm
/cc @castrojo or @parispittman for approval

**Comment by guineveresaenger** [comment]
lol k8s bot 😂

File: contributors/guide/release-notes.md [hunk 1]
```diff
@@ -30,4 +30,4 @@ For pull requests that don't need to be mentioned at release time, use the `/rel
 
 To see how to format your release notes, view the kubernetes/kubernetes [pull request template](https://git.k8s.io/kubernetes/.github/PULL_REQUEST_TEMPLATE.md) for a brief example. Pull Request titles and body comments can be modified at any time prior to the release to make them friendly for release notes.
 
-Release notes apply to pull requests on the master branch. For cherry-pick pull requests, see the [cherry-pick instructions](contributors/devel/cherry-picks.md). The only exception to these rules is when a pull request is not a cherry-pick and is targeted directly to the non-master branch.  In this case, a `release-note-*` label is required for that non-master pull request.
+Release notes apply to pull requests on the master branch. For cherry-pick pull requests, see the [cherry-pick instructions](../devel/cherry-picks.md). The only exception to these rules is when a pull request is not a cherry-pick and is targeted directly to the non-master branch.  In this case, a `release-note-*` label is required for that non-master pull request.
```
Comment (cblecker) [inline contributors/guide/release-notes.md]: Instead of using a relative link here, please add a leading `/`

**Comment by justaugustus** [comment]
Bump

**Comment by justaugustus** [comment]
Closing in favor of #3117.
(Commit is still attributed to @simonswine.)

**Pull request closed without merging**